Sources/NIOExtras/JSONRPCFraming+ContentLengthHeader.swift

```diff
@@ -0,0 +1,194 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2019 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+
+extension NIOJSONRPCFraming {
```

Comment:
@tomerd I nested this in `NIOJSONRPCFraming` so you can add your codecs into this too so we don't pollute the name-space too much.